packages/react/src/box/styles.ts

```diff
@@ -385,14 +385,24 @@ export const linkStyles = {
 };
 
 type FocusProps = Partial<{
-	/** If true, the focus indicator will be shown when the element receives keyboard focus. */
+	/** @deprecated use focusFor="keyboard". */
 	focus: boolean;
+	/** Display a focus indicator when the element receives focus. 'all' includes programmatic focus and 'keyboard' is for keyboard only focus*/
+	focusFor: 'all' | 'keyboard';
```

Comment:
Need to merge main and fix the conflicts. Interestingly, the PR preview still has the old props documented 
<img width="748" alt="image" src="https://github.com/agriculturegovau/agds-next/assets/853552/8db18b09-6b83-4a7c-8479-4b15f89035a6">